sr_multi_moveit/sr_multi_moveit_config/scripts/generate_robot_srdf.py

```diff
@@ -241,12 +241,7 @@ def __init__(self, description_file=None, load=True):
                     self.add_bimanual_arm_groups(self.robot.manipulators[0].arm.internal_name,
                                                  self.robot.manipulators[1].arm.internal_name,
                                                  False)
-                self.add_comments(comment)
-            if self.robot.manipulators[0].has_hand and self.robot.manipulators[1].has_hand:
```

Comment:
what is this deleted?

Reply:
@beatrizleon somehow I thought that was a redundant code repetition. Thanks for catching that, fixed now.